### PR TITLE
[full-ci] Support audio formats in `MediaViewer` App & rename it to `Preview`

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Renaming the mediaviewer app to `preview` and adding support for audio playback.
Also added support for overwriting the label of an editor's "Open with" action so that we can display `Preview` instead of `Open in Preview`.
While renaming the mediaviewer to preview is a breaking change, I've added alias handling with deprecation warnings so that configuration changes can happen over time.

## Related Issue

- Fixes https://github.com/owncloud/ocis/issues/1106

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:
- [x] rename mediaviewer to `Preview`
- [x] file action name should not be `Open in Preview` but only `Preview`
- [x] revert to hardcoded formats for audio, video, image (but revisit supported formats, especially for audio and don't use file extensions anymore but mimetypes instead)
- [x] icon for `Preview` -> `eye`
- [x] check if we can change pdf viewer icon to the file type icon

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10Files4</strong> https://drone.owncloud.com/owncloud/web/23221/19/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by dschmidt** [comment]
Does this fix https://github.com/owncloud/web/issues/5273?

**Comment by kulmann** [comment]
As noticed by @pascalwengerter opening `preview` for a file in a project-space or in a share now fails and complains that the `file.mimeType` property doesn't exist. Checking that next..

**Comment by kulmann** [comment]
> As noticed by @pascalwengerter opening `preview` for a file in a project-space or in a share now fails and complains that the `file.mimeType` property doesn't exist. Checking that next..

Turns out that this was not related to the file context but only happened if the current folder had a subfolder in it. Folders have `undefined` as mimetype. Fixed in https://github.com/owncloud/web/pull/6514/commits/848795b857fd0cd97828cc801b80f12b5dec5ffe
cc @pascalwengerter

**Review by pascalwengerter — APPROVED** [review]
Works like a charm!

File: changelog/unreleased/change-rename-mediaviewer [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Change: Rename mediaviewer to preview
+
+We've renamed the media-viewer app to preview because that describes the purpose of the app better and doesn't mislead users into thinking that it's a full blown media viewer.
+
+For the time being we've added an app alias handling to ownCloud Web which prints a deprecation warning if the preview app is tried to be loaded as `media-viewer`.
```
[the remark this answers is not in the document]
Comment (kulmann) [reply changelog/unreleased/change-rename-mediaviewer]: changed it to `enhancement`, thanks for the hint

**Pull request merged**